docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -44,18 +55,20 @@ image::images/data-qual-dash-detail.png[An expanded index with some failed resul
 
 
 [discrete]
-== Track unsuccessful mappings
+== Export data quality results
+
+You can share data quality results to help track your team's remediation efforts. First, follow the instructions under *Check indices* to generate results, then either:
```

Comment:
The way this rendered, it wasn't immediately clear what the two options are. Part of the problem is trying to create unofficial subheadings with bold text; I think bullets would work better, because they inherently convey a list of options. 

There's also a problem with "follow the instructions under **Check indices**," which should be a proper cross-ref link (which means the **Check indices** section above also needs an anchor ID to reference, which it doesn't currently have).

Reply:
Could this section be combined with the "Check indices section" or moved under that section as an L3 header?

Reply:
Thanks you two! Made a change. LMK if you think further changes are in order :)